server/googleActions.js

```diff
@@ -81,13 +96,11 @@ const GoogleActions = {
 
 	listGoogleBrews : async (req, res)=>{
 
-		oAuth2Client = GoogleActions.authCheck(req.account, res);
+		const oAuth2Client = GoogleActions.authCheck(req.account, res);
 
 		//TODO: Change to service account to allow non-owners to view published files.
 		// Requires a driveId parameter in the drive.files.list command
-		// const keys = JSON.parse(config.get('service_account'));
-		// const auth = google.auth.fromJSON(keys);
-		// auth.scopes = ['https://www.googleapis.com/auth/drive'];
+		// Then remove the `auth` parameter from the drive object initialization
```

Comment:
Lets delete this. With Stubs coming we don't need to make this a Service Account (see my rant under `listGoogleBrews` on my [other comment](https://github.com/naturalcrit/homebrewery/pull/2026#issuecomment-1042019886))